recipes/conversation_view/feed_item_row/feed_item_row_default.story.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <ul>
+    <dt-recipe-feed-item-row
+      :show-header="showHeader"
+      :avatar-image-url="avatarImageUrl"
+      :display-name="displayName"
+      :time="time"
+      :short-time="shortTime"
+      :is-active="isActive"
+      @on-hover="onHover"
+      @on-focus="onFocus"
```

Comment:
by convention should be `@hover="onHover"` `@focus="onFocus"`

Reply:
oh the JIRA mentioned onHover, onFocus events, made the changes as suggested. works now.